apps/preview/next/layouts/Showcases.tsx

```diff
@@ -157,7 +181,7 @@ export default function ShowcaseLayout({ children }: { children: ReactElement })
         </div>
       ) : null}
       <div className="e-page">
-        <div className="e-page-component">{children}</div>
+        <div className={classNames('e-page-component', noPaddings && 'e-page-component--no-paddings')}>{children}</div>
```

Comment:
https://github.com/JedWatson/classnames#usage
```suggestion
        <div className={classNames('e-page-component', { 'e-page-component--no-paddings': noPaddings })}>{children}</div>
```

Reply:
using classNames as I did is also valid, but I will change it